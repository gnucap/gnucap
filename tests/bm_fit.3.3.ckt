# FIT, vsrc test, with BC mismatch at start
v1 1 0 fit 1n,3  2n,3.5  3n,4  4n,4.5  5n,5  6n,5  7n,5  8n,5  9n,5
+ order=3 below=0 above=0
.print tran v(nodes)
.tran 0 10n .1n trace all
*>.status notime
.end
