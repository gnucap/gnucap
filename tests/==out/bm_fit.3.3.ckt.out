# FIT, vsrc test, with BC mismatch at start 
#Time       v(1)      
 0.         3.        
 1.p        3.        
 100.p      3.        
 200.p      3.        
 300.p      3.        
 400.p      3.        
 500.p      3.        
 600.p      3.        
 700.p      3.        
 800.p      3.        
 900.p      3.        
 1.n        3.        
 1.1n       3.0083    
 1.2n       3.0316    
 1.3n       3.0678    
 1.4n       3.1147    
 1.5n       3.1702    
 1.6n       3.2321    
 1.7n       3.2981    
 1.8n       3.3663    
 1.9n       3.4343    
 2.n        3.5       
 2.1n       3.5617    
 2.2n       3.6194    
 2.3n       3.6737    
 2.4n       3.7251    
 2.5n       3.774     
 2.6n       3.821     
 2.7n       3.8666    
 2.8n       3.9113    
 2.9n       3.9556    
 3.n        4.        
 3.1n       4.045     
 3.2n       4.0907    
 3.3n       4.1373    
 3.4n       4.1849    
 3.5n       4.2337    
 3.6n       4.2838    
 3.7n       4.3354    
 3.8n       4.3885    
 3.9n       4.4433    
 4.n        4.5       
 4.1n       4.5585    
 4.2n       4.6179    
 4.3n       4.6772    
 4.4n       4.7353    
 4.5n       4.7911    
 4.6n       4.8437    
 4.7n       4.892     
 4.8n       4.9348    
 4.9n       4.9711    
 5.n        5.        
 5.1n       5.0206    
 5.2n       5.0338    
 5.3n       5.0405    
 5.4n       5.042     
 5.5n       5.0392    
 5.6n       5.0333    
 5.7n       5.0253    
 5.8n       5.0164    
 5.9n       5.0076    
 6.n        5.        
 6.1n       4.9945    
 6.2n       4.991     
 6.3n       4.9891    
 6.4n       4.9888    
 6.5n       4.9895    
 6.6n       4.9911    
 6.7n       4.9932    
 6.8n       4.9956    
 6.9n       4.998     
 7.n        5.        
 7.1n       5.0015    
 7.2n       5.0024    
 7.3n       5.0029    
 7.4n       5.003     
 7.5n       5.0028    
 7.6n       5.0023    
 7.7n       5.0018    
 7.8n       5.0011    
 7.9n       5.0005    
 8.n        5.        
 8.1n       4.9996    
 8.2n       4.9994    
 8.3n       4.9994    
 8.4n       4.9994    
 8.5n       4.9994    
 8.6n       4.9996    
 8.7n       4.9997    
 8.8n       4.9999    
 8.9n       5.        
 9.n        5.        
 9.1n       5.        
 9.2n       5.        
 9.3n       5.        
 9.4n       5.        
 9.5n       5.        
 9.6n       5.        
 9.7n       5.        
 9.8n       5.        
 9.9n       5.        
 10.n       5.        
Gnucap   System status
iterations: op=0, dc=0, tran=254, fourier=0, total=356
transient timesteps: accepted=102, rejected=0, total=102
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
